src/dehtml.rs

```diff
@@ -23,11 +23,13 @@ struct Dehtml {
     /// Everything between <div name="quote"> and <div name="quoted-content"> is usually metadata
     /// If this is > `0`, then we are inside a `<div name="quoted-content">`.
     divs_since_quoted_content_div: u32,
+    /// All-Inkl just puts the quote into `<blockquote> </blockquote>`.
```

Comment:
This comment, and a link to the actual domain of All-Inkl (is it all-inkl.com?) is better put into `test_allinkl_blockquote`. Comment for the field does not describe what it contains. Is it the number of blockquotes nested inside a `blockquote` tag?

Reply:
I did this to match the comment on `divs_since_quote_div` above, which says that it's GMX. But I put another comment into the test and expanded this comment here.

Reply:
`divs_since_quote_div` also does not tell what this variable is for, it's just not part of this PR.

Reply:
Feel free to merge this as-is, I just commented on the comment because the code LGTM :)